src/summary/patient-chart-summary.component.tsx

```diff
@@ -1,17 +1,49 @@
 import React from "react";
 import { RouteComponentProps } from "react-router";
 import styles from "./patient-chart-summary.css";
-import HistorySection from "./history/history-section.component";
-import DocumentationSection from "./documentation/documentation-section.component";
+import DimensionsCard from "./documentation/dimensions-card.component";
+import VitalsCard from "./documentation/vitals-card.component";
+import ConditionsCard from "./history/conditions-card.component";
+import AllergyCard from "./history/allergy-card.component";
+import NotesCard from "./history/notes-card.component";
+import ProgramsCard from "./history/programs/programs-card.component";
 
 export default function PatientChartSummary(props: PatientChartSummaryProps) {
+  const config = [
+    "conditions",
+    "programs",
+    "allergies",
+    "notes",
+    "vitals",
+    {
+      module: DimensionsCard
```

Comment:
why not just stick to one type in the array?

Reply:
+1, I don't want to try and make esm-module-config accept polymorphic arrays

Reply:
See above. How to represent cases in which widgets are from outside this repo?

Reply:
For now, I am going to leave as is. Per  @brandones, he still needs to add funcationlity to the config module for arrays. Additionally, he and I were discussing how to handle dynamic loading of modules referenced in config. I think it would be easiest if the config module could handle the systemjs calls (and handling promises) rather than forcing the module which is doing the import to handle that. Curious to here @joeldenning's take on this.